packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -2,35 +2,14 @@ import Ceramic from '../ceramic'
 import { Ed25519Provider } from 'key-did-provider-ed25519'
 import { AnchorStatus, Doctype, IpfsApi } from "@ceramicnetwork/common"
 import tmp from 'tmp-promise'
-import IPFS from 'ipfs-core'
 import * as u8a from 'uint8arrays'
-
-import getPort from 'get-port'
-
-import dagJose from 'dag-jose'
-import basicsImport from 'multiformats/cjs/src/basics-import.js'
-import legacy from 'multiformats/cjs/src/legacy.js'
+import * as _ from 'lodash'
```

Comment:
this seems potentially confusing.  Is this a common convention in javascript code to import a package as `_`?  Wouldn't it be clearer just to call it `lodash`?

Reply:
It is a convention. Comes from times of a package called [underscore](https://underscorejs.org). Lodash is effectively a replacement for underscore, that's why "_".

Reply:
Ah okay.  Yeah I guess this package provides such fundamental utilities it's reasonable to give it someone special treatment.  Sounds good

Reply:
At the time it was groundbreaking indeed. Removing it away.

Reply:
Was that sarcasm in "fundamental"?

Reply:
No, it wasn't sarcastic, I was being genuine.  It seems like this library is designed to give some pretty basic core primitives that are generally useful and can almost be thought of as extensions to the language's standard library.